pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = --pep8
```

Comment:
is this meant to be `adopts`

Reply:
https://docs.pytest.org/en/latest/customize.html?highlight=addopts

I'm not sure why its `addopts`; I thought it was a typo at first too.